projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -22,6 +22,13 @@ function wpcom_launchpad_get_task_definitions() {
 				add_action( 'load-site-editor.php', 'wpcom_track_edit_site_task' );
 			},
 		),
+		// Design completed checks for task completion while design_select will always return true
```

Comment:
```suggestion
		// design_completed checks for task completion while design_selected always returns true.
```
Sorry, this is a nitpick, but putting a period at the end of the comment will make phpcs happy.

Reply:
Thanks @DustyReagan!